src/lib/pipeline/FetchUnit.cc

```diff
@@ -98,6 +98,7 @@ void FetchUnit::tick() {
                 blockSize_ - bufferOffset);
```

Comment:
All the added newlines can be removed. I assume they're left over from previously removed developments

Reply:
Removed most. Yes, added during old developments